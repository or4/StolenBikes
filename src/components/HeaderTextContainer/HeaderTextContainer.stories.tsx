import { storiesOf } from '@storybook/react';
import * as React from 'react';
import { HeaderTextContainer } from './HeaderTextContainer';

storiesOf('HeaderTextContainer', module).add('HeaderTextContainer', () => <HeaderTextContainer />);
