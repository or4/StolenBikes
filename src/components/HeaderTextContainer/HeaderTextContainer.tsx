import React from 'react';
import styled from 'styled-components';
import { media } from 'ui/media';

const Container = styled.div``;

const Title = styled.div`
    font-weight: 600;

	${media.mobile`
    	font-size: 18px;
  	`}

	${media.tablet`
    	font-size: 32px;
	`}

    ${media.desktop`
    	font-size: 48px;
	`}
`;

const Description = styled.div`
	${media.mobile`
    	font-size: 14px;
  	`}

	${media.tablet`
    	font-size: 16px;
	`}

    ${media.desktop`
    	font-size: 32px;
	`}
`;

export const HeaderTextContainer = () => (
    <Container>
        <Title>Police Department of Berlin</Title>
        <Description>Stolen bykes</Description>
    </Container>
);
