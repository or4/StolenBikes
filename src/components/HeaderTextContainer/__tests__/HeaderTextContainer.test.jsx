import React from 'react';
import { mount } from 'enzyme';
import { HeaderTextContainer } from '../HeaderTextContainer';

describe('HeaderTextContainer', () => {
    it('should render correctly', () => {
        const button = <HeaderTextContainer />;

        const mounted = mount(button);
        expect(mounted).toMatchSnapshot();
        mounted.unmount();
    });
});
