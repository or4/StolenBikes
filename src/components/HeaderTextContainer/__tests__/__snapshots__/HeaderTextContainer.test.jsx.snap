// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderTextContainer should render correctly 1`] = `
<HeaderTextContainer>
  <styled.div>
    <StyledComponent
      forwardedComponent={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": true,
            "lastClassName": "bDWFJH",
            "rules": Array [],
          },
          "displayName": "styled.div",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": "div",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
    >
      <div
        className="sc-bdVaJa bDWFJH"
      >
        <styled.div>
          <StyledComponent
            forwardedComponent={
              Object {
                "$$typeof": Symbol(react.forward_ref),
                "attrs": Array [],
                "componentStyle": ComponentStyle {
                  "componentId": "sc-bwzfXH",
                  "isStatic": true,
                  "lastClassName": "keAcTh",
                  "rules": Array [
                    "
    font-weight: 600;

	",
                    "
        @media ",
                    "(max-width: ",
                    "479",
                    "px)",
                    " {
            ",
                    "
    	font-size: 18px;
  	",
                    "
        }
    ",
                    "

	",
                    "
        @media ",
                    "(min-width: ",
                    "480",
                    "px) and (max-width: ",
                    "991",
                    "px)",
                    " {
            ",
                    "
    	font-size: 32px;
	",
                    "
        }
    ",
                    "

    ",
                    "
        @media ",
                    "(min-width: ",
                    "992",
                    "px)",
                    " {
            ",
                    "
    	font-size: 48px;
	",
                    "
        }
    ",
                    "
",
                  ],
                },
                "displayName": "styled.div",
                "foldedComponentIds": Array [],
                "render": [Function],
                "styledComponentId": "sc-bwzfXH",
                "target": "div",
                "toString": [Function],
                "warnTooManyClasses": [Function],
                "withComponent": [Function],
              }
            }
            forwardedRef={null}
          >
            <div
              className="sc-bwzfXH keAcTh"
            >
              Police Department of Berlin
            </div>
          </StyledComponent>
        </styled.div>
        <styled.div>
          <StyledComponent
            forwardedComponent={
              Object {
                "$$typeof": Symbol(react.forward_ref),
                "attrs": Array [],
                "componentStyle": ComponentStyle {
                  "componentId": "sc-htpNat",
                  "isStatic": true,
                  "lastClassName": "cxvLWi",
                  "rules": Array [
                    "
	",
                    "
        @media ",
                    "(max-width: ",
                    "479",
                    "px)",
                    " {
            ",
                    "
    	font-size: 14px;
  	",
                    "
        }
    ",
                    "

	",
                    "
        @media ",
                    "(min-width: ",
                    "480",
                    "px) and (max-width: ",
                    "991",
                    "px)",
                    " {
            ",
                    "
    	font-size: 16px;
	",
                    "
        }
    ",
                    "

    ",
                    "
        @media ",
                    "(min-width: ",
                    "992",
                    "px)",
                    " {
            ",
                    "
    	font-size: 32px;
	",
                    "
        }
    ",
                    "
",
                  ],
                },
                "displayName": "styled.div",
                "foldedComponentIds": Array [],
                "render": [Function],
                "styledComponentId": "sc-htpNat",
                "target": "div",
                "toString": [Function],
                "warnTooManyClasses": [Function],
                "withComponent": [Function],
              }
            }
            forwardedRef={null}
          >
            <div
              className="sc-htpNat cxvLWi"
            >
              Stolen bykes
            </div>
          </StyledComponent>
        </styled.div>
      </div>
    </StyledComponent>
  </styled.div>
</HeaderTextContainer>
`;
